src/rdo.rs

```diff
@@ -1897,18 +1896,19 @@ pub fn rdo_loop_decision<T: Pixel>(
     lru_h[pli] = sb_h / (1 << sb_v_shift);
   }
 
-  let mut best_index = vec![-1; sb_w * sb_h];
-  let mut best_lrf = ArrayVec::<[Vec<RestorationFilter>; 3]>::new();
+  // Based on `RestorationState::new`
+  const MAX_SB_SHIFT: usize = 4;
```

Comment:
Can we keep those in a single place?

Reply:
You mean make these available more generally? I'm not sure if these variables as they're defined here are useful outside the context of this function. It doesn't seem so, looking at usages of `ts.restoration.planes[pli].rp_cfg.sb_h_shift`.